customercashbalancetransaction.go

```diff
@@ -24,6 +24,7 @@ type CustomerCashBalanceTransactionType string
 const (
 	CustomerCashBalanceTransactionTypeAppliedToPayment     CustomerCashBalanceTransactionType = "applied_to_payment"
 	CustomerCashBalanceTransactionTypeFunded               CustomerCashBalanceTransactionType = "funded"
+	CustomerCashBalanceTransactionTypeFundingReversed      CustomerCashBalanceTransactionType = "funding_reversed"
```

Comment:
Also not in Changelog, for now we can manually add
`Add support for new value `funding_reversed` on enum CustomerCashBalanceTransactionType`